src/elements/content-sidebar/__tests__/MetadataSidebarRedesigned.test.tsx

```diff
@@ -1,23 +1,75 @@
 import React from 'react';
-import { IntlProvider } from 'react-intl';
+import { screen } from '@testing-library/react';
+import MetadataSidebarRedesign from '../MetadataSidebarRedesign';
+import { FIELD_PERMISSIONS_CAN_UPLOAD } from '../../../constants';
+import { render } from '../../../test-utils/testing-library';
 
-import { screen, render } from '@testing-library/react';
+jest.unmock('react-intl');
```

Comment:
this can be removed since youre importing testing-library

Reply:
I double checked this, apparently `jest.unmock` does not reach the shared-feature when the `__mock__` is mocking `react-intl`

so you can leave the `jest.unmock` where it is but put a comment on why its there and why it cant be removed yet

I would also swap out for the Component instead of the default Component so you dont have to mock the HOCs
What I have below should work

```
import React from 'react';
import { MetadataSidebarRedesignComponent as MetadataSidebarRedesign } from '../MetadataSidebarRedesign';
import { FIELD_PERMISSIONS_CAN_UPLOAD } from '../../../constants';
import { render, screen } from '../../../test-utils/testing-library';

jest.unmock('react-intl'); // ADD COMMENT HERE

const mockAPI = {
    getFile: jest.fn((id, successCallback) => {
        successCallback({ id, [FIELD_PERMISSIONS_CAN_UPLOAD]: true });
    }),
    getMetadata: jest.fn((file, successCallback) => {
        successCallback({ editors: [], templates: [] });
    }),
};
const api = {
    getFileAPI: jest.fn().mockReturnValue(mockAPI),
    getMetadataAPI: jest.fn().mockReturnValue(mockAPI),
};

describe('elements/content-sidebar/Metadata/MetadataSidebarRedesign', () => {
    const renderComponent = (props = {}) => {
        render(<MetadataSidebarRedesign {...props} />);
    };
```